### DISCUSSION TITLE
我忘记备份了，但是有本地文件，怎么恢复回去呢

### DISCUSSION THREAD

**Discussion by rabbit-yu** [halo-dev/halo#5713] [Q&A]
### System information

我迁移服务器，没有备份，只有个本地文件夹，我想用docker重新启动一个，怎么恢复回去呢

### What is the project operation method?

Docker

### What happened?

文件结构：db.json  public  plugins source themes _config.landscape.yml  _config.yml scaffolds

**Comment by JohnNiang** [comment]
首先，需要确认你是否用的 Halo。Halo 目前没有这样的目录结构。

另外，你需要说明使用的哪个 Halo 版本，以及你已经做过的尝试。
